api/service/restclientsupport/service.go

```diff
@@ -259,3 +260,8 @@ func (s *Service) NotifyService(params map[string]interface{}) {
 func (s *Service) SetMessageChan(messageChan chan *msg_pb.Message) {
 	s.messageChan = messageChan
 }
+
+// APIs for the services.
+func (s *Service) APIs() []rpc.API {
```

Comment:
All the APIs are nil. I don't think it's necessary to add this interface.

Reply:
my understanding is all services can provide RPC service, via this interface.  sounds like a good framework.

Reply:
Yes, it is just a framework, so the services can provide API.

Reply:
I can see your point. But I don't think it's needed in this PR -- kinda premature to add it.
* If we found we never add APIs in this way these interfaces are dead code. 
* If we need to add APIs to some service, why not add it by then.